alacritty_terminal/tests/ref/underline/size.json

```diff
@@ -1 +1 @@
-{"width":939.0,"height":503.0,"cell_width":8.0,"cell_height":16.0,"padding_x":5.0,"padding_y":3.0,"cols":116,"screen_lines":31}
+{"width":620.0,"height":500.0,"cell_width":8.0,"cell_height":20.0,"padding_x":0.0,"padding_y":0.0,"screen_lines":25,"cols":77}
```

Comment:
The size is small and stuff is still rendered nicely without linebreaks or anything like that. That's exactly how it should be, thanks.